demisto_sdk/commands/common/constants.py

```diff
@@ -524,6 +524,9 @@ class FileType(Enum):
 WIZARD_DIR_REGEX = fr'{PACK_DIR_REGEX}\/{WIZARDS_DIR}'
 WIZARD_JSON_REGEX = fr'{WIZARD_DIR_REGEX}\/wizard-([^/]+)\.json'
 
+RELATIVE_HREF_URL_REGEX = r'(<.*?href\s*=\s*"((?!(?:https?:\/\/)|#|(?:mailto:)).*?)")'
+RELATIVE_MARKDOWN_URL_REGEX = r'^(?![!])(\[.*?\])\(((?!(?:https?:\/\/)|#|(?:mailto:)).*?)\)$'
```

Comment:
why ^ and &? only if this takes up the whole line?

Reply:
also, looks like 2 links on 1 line is a bit buggy, 
`(?![!])(\[[^\[]*\])\(((?!(?:https?:\/\/)|#|(?:mailto:)).*?)\)` without multiline seems like it works a bit better, lets discuss

Reply:
As we discussed, removed those :)